index.html

```diff
@@ -7,20 +7,21 @@
 <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Inter">
 </head>
 
-<body> 
+<body > 
 
-  <section class="navigation">
-    <div class="toolbar-container">
-        <div class="logo">Welcome</div>
+  <header class="navigation">
+    <nav class="toolbar-container">
+        <div class="logo"><a href="#welcome">Welcome</a></div>
         <div class="menu-bar">
           <a href="#portoflio">Portfolio</a>
           <a href="#about">About</a>
           <a href="#contact">Contact</a>
         </div> 
-        <a class="icon" >
-            <i class="fa fa-bars"></i>
-      </div>
-    </section>
+        <a class="icon" ><a href="#menu-bar">
+            <i class="fa fa-bars"></i></a>
```

Comment:
- [Optional] Please close all HTML tags. It looks like one of your `<a>` tags is not close. Kindly add a closing tag.